Sources/Views/KMTagLabelsView.swift

```diff
@@ -0,0 +1,103 @@
+//
+//  KMTagLabelsView.swift
+//  KommunicateChatUI-iOS-SDK
+//
+//  Created by sathyan elangovan on 21/11/22.
+//
+
+import Foundation
+
+class KMTagLabelsView: UIView {
+
+    var tagNames: [String] = [] {
+        didSet {
+            addTagLabels()
+        }
+    }
+    
+    let tagHeight:CGFloat = 30
+    let tagPadding: CGFloat = 16
+    let tagSpacingX: CGFloat = 8
+    let tagSpacingY: CGFloat = 8
```

Comment:
A struct would be better than creating these variables if are just reading them.

Reply:
used struct to read